ch6/03.scm

```diff
@@ -2,144 +2,60 @@
 
 ;; 1. (lambda (x y) (p (+ 8 x) (q y)))
 ;; ==>
-(lambda (x y cont) (q y (lambda(v)
-			  (p v (cont (+ 8 x cont))))))
+(lambda (x y cont)
+  (q y (lambda (val)
+	 ((lambda (val2) (p val2 val cont))
+	  (+ 8 x)))))
 
```

Comment:
Yes, you are right! I forgot '+' isn't a normal procedure call! Same as below. I will correct my PR soon.